src/main/java/com/synopsys/integration/detect/workflow/componentlocationanalysis/GenerateComponentLocationAnalysisOperation.java

```diff
@@ -54,12 +56,16 @@ public void locateComponents(Set<Component> componentsSet, File scanOutputFolder
         }
```

Comment:
What do we expect to see in the results (console and status.json) when CLA fails?

Reply:
There will not be a results entry for CLA in neither console nor status.json in such a case.
This is how some other result-producing things behave.
Additionally, the `Detect Result` section only gets displayed when there is at least one entry that belongs to it.